static/src/javascripts/__flow__/types/ab-tests.js

```diff
@@ -43,7 +43,7 @@ declare type EpicVariant = Variant & {
     ) => string,
     ctaText?: string,
     secondaryCta?: EpicCta,
-    copy?: AcquisitionsEpicTemplateCopy,
```

Comment:
This model is now only used for liveblog epics, and the copy should always be provided (we're not using google sheets anymore)